source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -33,12 +33,24 @@ You can create a Prometheus service within your PaaS space and allow it to bind
   - `cf create-service gds-prometheus prometheus <service-instance-name>`
 2. Either [update your app's manifest.yml] to bind your new service or bind it using the CLI
   - `cf bind-service <app-name> <service-instance-name>`
+3. If your app uses a custom domain, you need to make sure that the [authorization header and other app headers][] are being forwarded otherwise the basic auth to the metrics endpoint will fail and your app won't have access to your other headers.
```

Comment:
'are being forwarded' where are they being forwarded to, the custom domain?

Reply:
Forward to the app, which is located at the URL given in PaaS